data/api/client-server/pushrules.yaml

```diff
@@ -237,7 +235,214 @@ paths:
                   }
       tags:
         - Push notifications
-  "/pushrules/{scope}/{kind}/{ruleId}":
+  /pushrules/global/:
+    get:
+      summary: Retrieve all push rules.
+      description: |-
+        Retrieve all push rules for this user.
+      operationId: getPushRulesGlobal
+      security:
+        - accessTokenQuery: []
+        - accessTokenBearer: []
+      responses:
+        "200":
+          description: All the push rules for this user.
+          content:
+            application/json:
+              schema:
+                $ref: definitions/push_ruleset.yaml
+              examples:
+                response:
+                  value: {
+                    "content": [
```

Comment:
I believe, if we move this example to `definitions/push_ruleset.yaml`, it will magically be used for both endpoints, without the need to duplicate it.